src/GovUk.Education.ExploreEducationStatistics.Common/Services/TableStorageService.cs

```diff
@@ -59,5 +61,10 @@ public async Task<TableResult> UpdateEntity(string tableName, ITableEntity entit
             var table = _client.GetTableReference(tableName);
             return await table.ExecuteAsync(TableOperation.InsertOrReplace(entity));
         }
+        
+        public CloudTable GetCloudTable(string tableName)
+        {
+            return _client.GetTableReference(tableName);
```

Comment:
This doesn't create the table if it doesn't exist like we do everywhere else, but as mentioned above you can use `GetTableAsync`.

Reply:
done